### ISSUE TITLE
Include more metrics to evaluate modes

### ISSUE THREAD

**Issue by ADPriceless** [ADPriceless/TDA_Assignment_FakeNews#2]
Add to training to view in history:
- Precision
- Recall
- Confusion matrix using comet: https://medium.com/swlh/debugging-classifiers-with-confusion-matrices-6210c77c5d65

Add metrics to test set:
- Confusion matrix
- sklearn classifier score (includes precision, recall, etc.)

Also investigate balanced accuracy score
